cdci_data_analysis/pytest_fixtures.py

```diff
@@ -702,6 +721,13 @@ def from_run_analysis_response(cls, r):
             session_id = r.json()['session_id'],
             job_id = r.json()['job_monitor']['job_id']
         )
+
+    @classmethod
```

Comment:
this largely duplicates previous method.
if you add this, the previous one may be hust `from_run_analysis_response(r.json())`